make_plant_parts_eia.py

```diff
@@ -1047,9 +1049,7 @@ def get_qualifiers(self,
             # restric the number of columns in here to only include the ones we
             # need, unlike get_consistent_qualifiers, dedup_on_category
             # preserves all of the columns from record_df
-            record_df = record_df[
-                [c for c in record_df.columns
-                 if c in base_cols + list(part_df.columns) + [record_name]]]
+            record_df = record_df[base_cols + [record_name]]
```

Comment:
this is basically all I had to change to fix the op status... my previous restriction of the columns was too encompassing